govcd/vm.go

```diff
@@ -850,3 +851,186 @@ func (vm *VM) SetGuestCustomizationSection(guestCustomizationSection *types.Gues
 
 	return vm.GetGuestCustomizationSection()
 }
+
+// AddInternalDisk creates disk type *types.DiskSettings to the VM.
+// Returns new disk ID and error.
+// Runs synchronously, VM is ready for another operation after this function returns.
+func (vm *VM) AddInternalDisk(diskData *types.DiskSettings) (string, error) {
+	err := vm.Refresh()
+	if err != nil {
+		return "", fmt.Errorf("error refresing vm: %s", err)
```

Comment:
I noticed you removed VM name from this and two more identical error messages:
https://github.com/vmware/go-vcloud-director/commit/f0aee2fde6614f85633967023aaf8c3e3714158e

How does the message look like now?

BTW, there's a typo in a all three.

Reply:
updated mistype. We can't have `vm.VM.Name,` as we can get nil errors  if VM is empty